src/Krank/Checkers/IssueTracker.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE ApplicativeDo #-}
+{-# LANGUAGE DataKinds #-}
```

Comment:
because of `Req.Https` which is a data constructor for the `Url a` data type.
I'm not sure I'm going at it the right way in term of type, but I did not want to accept a `Url a` that could be just `Http`...